cnxarchive/sql/search/query.sql

```diff
@@ -64,6 +64,7 @@ FROM
   derived_weighted_query_results AS wqr
 WHERE
   wqr.module_ident = lm.module_ident
+  AND lm.portal_type != 'CompositeModule'
```

Comment:
I guess we can't really search composite module content at the moment, but is there any reason not to search its metadata? Perhaps that's the problem, that the query breaks in the full text search part.

Perhaps we should note this failing somewhere?